components/kyma-environment-broker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build image
+FROM golang:1.13-alpine AS build
+
+WORKDIR /go/src/github.com/kyma-incubator/compass/components/kyma-environment-broker
+
+COPY cmd cmd
+COPY internal internal
+COPY vendor vendor
+
+RUN CGO_ENABLED=0 go build -o /bin/kyma-env-broker ./cmd/broker/main.go
+
+# Get latest CA certs
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+# Final image
+FROM scratch
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=build /bin/kyma-env-broker /bin/kyma-env-broker
+
+ENV APP_PORT 8080
```

Comment:
When you specify the env variable with a port that can be overridden during the run phase, why you then use the `EXPOSE` keyword that always will show 8080/tcp?